### PR TITLE
Add User Agents

### PR DESCRIPTION
**Pull request by svix-frank**

### CODE REVIEW

File: go/internal/version/version.go [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package version
+
+const Version = "0.25.0"
```
Comment (svix-frank) [inline go/internal/version/version.go:3]: there's no other way to inject this in go

File: java/lib/src/main/java/com/svix/Svix.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import com.svix.internal.auth.HttpBearerAuth;
 
 public final class Svix {
+	public static final String VERSION = "0.25.0";
```
Comment (svix-frank) [inline java/lib/src/main/java/com/svix/Svix.java:8]: or java

File: javascript/src/index.ts [hunk 3]
```diff
@@ -38,11 +41,22 @@ import * as base64 from "@stablelib/base64";
 import * as sha256 from "fast-sha256";
 
 const WEBHOOK_TOLERANCE_IN_SECONDS = 5 * 60; // 5 minutes
+const VERSION = "0.25.0";
```
Comment (svix-frank) [inline javascript/src/index.ts:44]: We don't use a packager so this I don't know of another option here but in the future we could use rollup/webpack and inject this via a plugin

File: python/svix/api.py [hunk 4]
```diff
@@ -68,6 +68,10 @@ def __init__(self, configuration: Configuration) -> None:
     @contextmanager
     def _api(self) -> t.Generator[ApiClass, None, None]:
         with ApiClient(self._configuration) as api_client:  # type: ignore
+            
+            import svix
+            api_client.user_agent = f"svix-libs/{svix.__version__}/python"
+
```
Comment (svix-frank) [inline python/svix/api.py:74]: Does this seem reasonable?

**Review by tasn — APPROVED** [review]
Can you just add a point on the readme/some process to make sure that we bump these too?

File: python/svix/api.py [hunk 5]
```diff
@@ -68,6 +68,11 @@ def __init__(self, configuration: Configuration) -> None:
     @contextmanager
     def _api(self) -> t.Generator[ApiClass, None, None]:
         with ApiClient(self._configuration) as api_client:  # type: ignore
+
+            import svix
```
Comment (tasn) [inline python/svix/api.py:72]:
```suggestion
            from . import __version__
```
Maybe? (make sure it works)

**Pull request merged**